src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Debugger_InProc.cs

```diff
@@ -33,5 +56,105 @@ public void SetBreakPoint(string fileName, int lineNumber, int columnIndex)
         public void ExecuteStatement(string statement) => _debugger.ExecuteStatement(statement);
 
         public Common.Expression GetExpression(string expressionText) => new Common.Expression(_debugger.GetExpression(expressionText));
+
+        /// <summary>
+        /// Executes the specified action delegate and retries if Operation Not Supported is thrown.
+        /// </summary>
+        /// <param name="action">Action delegate to exectute.</param>
+        private void WaitForRaiseDebuggerDteCommand(Action action)
+        {
+            bool actionSucceeded = false;
+
+            Func<bool> predict = delegate
+            {
+                try
+                {
+                    action();
+                    actionSucceeded = true;
+                }
+                catch (COMException ex)
+                {
+                    if ((uint)ex.ErrorCode != OperationNotSupportedHResult)
+                    {
+                        string message = string.Format(
+                            CultureInfo.InvariantCulture,
+                            "Failed to raise debugger command, an unexpected '{0}' was thrown with the HResult of '{1}'.",
+                            typeof(COMException),
+                            ex.ErrorCode);
+
+                        throw new Exception(message, ex);
+                    }
+
+                    actionSucceeded = false;
+                }
+
+                return actionSucceeded;
+            };
+
+            // Repeat the the command if "Operation Not Supported" is thrown.
```

Comment:
>the [](start = 22, length = 3)

typo

Reply:
Thanks!

---
In reply to: [136207355](https://github.com/dotnet/roslyn/pull/21824#discussion_r136207355) [](ancestors = 136207355)